src/main/java/frc/robot/poseestimator/limelights/LimelightsHardware.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.poseestimator.limelights;
+
+import frc.utils.GBSubsystem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LimelightsHardware extends GBSubsystem {
```

Comment:
why subsystem

Reply:
because it functions as a subsystem for the limelights

Reply:
It does not need to be a subsystem